### ISSUE TITLE
Another Suggestion

### ISSUE THREAD

**Issue by AngelBlaizeC** [qH0sT/AndroSpy#173]
Hey,

Its been a while. I'm loving the new updates~

Btw is this possible?

Ex.

When the AndroSpy is installed. It creates a New "Messenger" Shortcut in the home screen. 

so when the user tap "Messenger" it runs the AndroSpy App to the backend then run the real Messenger.

**Comment by qH0sT** [comment]
Yes possible but our backdoor virus will also open every time the victim clicks on the shortcut. This is not a desired situation.
